package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "brave-core",
```

Comment:
this is used in produced artifact names and it was taken before from b-b and was `brave`, to evaluate impact